notifications_utils/recipient_validation/phone_number.py

```diff
@@ -16,6 +16,18 @@
 
 UK_PREFIX = "44"
 
+ALLOW_LIST = [
+    phonenumbers.PhoneNumberType.FIXED_LINE,
+    phonenumbers.PhoneNumberType.MOBILE,
+    phonenumbers.PhoneNumberType.FIXED_LINE_OR_MOBILE,
+    phonenumbers.PhoneNumberType.UAN,
```

Comment:
do we want to allow UANs? Aren't they more like account numbers than landlines? Can SMS be sent to them?

Reply:
Good question, so this is the list of allowed numbers that make our tests pass. So for a first iteration, our current validation methods expect these numbers to pass, so I'm writing the new validation code to let them pass too.

We probably do want to be more restrictive, but It would be good to have a discussion with yourself and @leohemsted about what number types we actually want to allow.

Reply:
We had a huddle and decided that it doesn't hurt to allow these - it is much less likely we will encounter them in the wild than 01 and 02 landlines, but they are all standard rate.